src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable {
+    private Colors color;
+    private double side;
+
+    public Figure(Colors color, double side) {
+        this.color = color;
+        this.side = side;
+    }
+
+    public double getSide() {
+        return side;
+    }
+
+    public Colors getColor() {
+        return color;
+    }
+
+    public abstract double getArea();
+}
```

Comment:
Don't use abstract classes to set behavior for classes. Consider using an interface to define the behavior and then implement it in your concrete classes.